### ISSUE TITLE
 A11Y-DUO: unable to tap on any incoming <elided app feature> control when <elided app> is playing

### ISSUE THREAD

**Issue by kf6gpe** [flutter/flutter#68823]
https://buganizer.corp.google.com/issues/170234410. Comments should go there, please.

**Comment by chunhtai** [comment]
issue seems to be fixed in the application, closing this issue
